backend/vendors.json

```diff
@@ -0,0 +1,206 @@
+{ "index": { "_index": "vendors", "_type": "vendor", "_id": "0" } }
```

Comment:
Please remove this file as it is not necessary.

Reply:
Got it. Already added these two tables to the schema and deleted the first piece of data in JSON